vignettes/finalize.Rmd

```diff
@@ -143,6 +143,16 @@ finalize_plot(plot = p,
               debug = TRUE)
 ```
 
+### Exporting a graph with no title or caption
+
+One special case of the top-level position adjustments is the elimination of the title and caption altogether. By setting `title_width = 0`, you can export a graph with just the plot and legend elements. Note that in this situation, the constant `margin_title_l` is no longer relevant. `margin_plot_l` is now the margin between the left side of the image and the beginning of the plot (See the next section for more details about these constants).
+
```

Comment:
Do we want to include a caption in this example, to show that it's possible? Or at least mention that captions are still supported with this option?